### PR TITLE
Improve bundle files

### PR DESCRIPTION
**Pull request by adela-istrate-hs**

### CODE REVIEW

**Comment by adela-istrate-hs** [comment]
@valentin-radulescu-hs @andrei-picus-hs can you review?

**Comment by valentin-radulescu-hs** [comment]
Maybe we could rename `bundled` to something like `build`? LGTM otherwise

File: webpack.config.js [hunk 1]
```diff
@@ -14,14 +15,14 @@ module.exports = {
       {test: /\.less$/, loader: ExtractTextPlugin.extract('style',
         'css!postcss!less')},
       // handles the bootstrap glyphicons
-      {test: /\.(png|woff|woff2|eot|ttf|svg)$/,
-        loader: 'url-loader?limit=100000&name=./statics/icons.svg'}
+      {test: /\.(woff|woff2|eot|ttf|svg)$/,
+        loader: 'file-loader?name=icons/icons.[ext]'}
```
Comment (andrei-picus-hs) [inline webpack.config.js]: Let's remove the `name` param to keep the original filenames.

File: webpack.config.js [hunk 2]
```diff
@@ -14,14 +15,14 @@ module.exports = {
...
     ]
   },
   postcss: [
     autoprefixer({browsers: ['last 2 versions']})
   ],
   plugins: [
-    new ExtractTextPlugin('./statics/style.css')
+    new ExtractTextPlugin('style.css')
```
Comment (andrei-picus-hs) [inline webpack.config.js]: Let's rename this to `bundle.css` to be in line with the JS bundle.

**Comment by andrei-picus-hs** [comment]
Finished review.

**Comment by adela-istrate-hs** [comment]
@andrei-picus-hs  @valentin-radulescu-hs  can I merge?

**Comment by andrei-picus-hs** [comment]
LGTM

**Comment by valentin-radulescu-hs** [comment]
LGTM

**Pull request merged**